src/components/LoginFields.js

```diff
@@ -14,8 +17,11 @@ const LoginFields = () => {
       name,
     }).unwrap()
       .then((response) => {
+        console.log(response);
```

Comment:
Kindly remove the console logs from your code since they are unnecessary for the main functionality of the app.